client/src/components/MorePlacesSection.jsx

```diff
@@ -48,7 +61,10 @@ const MorePlacesSection = () => {
 
   return (
     <MorePlacesSectionContainer>
-      <SectionHeader page={page} pages={places.length / perPage} updatePage={updatePage} />
+      <InnerSectionContainer>
+        <SectionHeader page={page} pages={places.length / perPage} updatePage={updatePage} />
+        <ItemPhoto isSaved={places[0].isSaved} isSuperhost={places[0].isSuperhost} thumbnailUrl={places[0].thumbnailUrl} />
```

Comment:
I'm sure you'll fix this later but just a reminder that I think this would return all the same places data (index 0) :)